html/about.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/about.css">
+    <title>About</title>
+</head>
+
+<body>
+
+    <div id="menu-list" class="mobile-menu-container">
+        <div id="menu-close" class="mobile-menu-close-btn"><a>&#x2715;</a></div>
+        <ul>
+            <li><a id="item-portfolio" href="../index.html">Home</a></li>
+            <li><a id="item-about" href="#">About</a></li>
+            <li><a id="item-program" href="#program">Program</a></li>
+            <li><a id="item-join" href="#join">Join</a></li>
+            <li><a id="item-sponsor" href="#sponsor">Sponsor</a></li>
+            <li><a id="item-news" href="#news">News</a></li>
+        </ul>
+    </div>
+
+    <header class="navbar">
+        <div class="top-header">
+            <ul>
+                <li><a class="fb-ic" href="#"><img src="../images/fb.png" alt="facebook"></a></li>
+                <li><a class="tw-ic" href="#"><img src="../images/tw.png" alt="twitter"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <nav class="nav-container">
+            <a class="logo" href="../index.html">Tech<span>Meet</span></a>
+            <img id="menu-button" class="menu-button" src="../images/menu.svg" alt="hamburger menu">
+            <div class="right-ctn">
+                <ul class="nav-menu">
+                    <li><a class="about" href="#">About</a> </li>
```

Comment:
## OPTIONAL
- kindly change this to index.html so the user can navigate easily to the Home page from the about page